### PR TITLE
Various small changes: comments, renamed methods and improved statements

### PR DESCRIPTION
**Pull request by ramzesenok**
### Improved the way comments are written:
- `override`s don't need description comments, they ignore them
- description comments are used only to describe the next variable/function/class/etc
- variables don't `return` values, they store them instead, therefore it's not absolutely correct to describe them as `/// Returns something`. I didn't change it everywhere, there are a lot, only a few. (IMO even computed properties are still variables, their computation is incapsulated)
- `// MARK: Blah-blah` should be `// MARK: - Blah-blah`. This is nice and it also helps Xcode (it will create a nice line in the code editor to help you visually separate things)

### Renamed few methods to more suitable names
- `cell.correctAnswer()`, `cell.wrongAnswer()` and `cell.unknownAnswer()` are not really saying immediately what are they doing, so I guess `cell.setCorrect/Wrong/UndefinedAnswerStyle()` suits better

### Improved some statements
- `switch-case` instead of `if` where it seemed better
- Other tiny changes

### CODE REVIEW

File: Unwrap/Extensions/String-Lines.swift [hunk 1]
```diff
@@ -9,7 +9,7 @@
 import Foundation
 
 extension String {
-    /// Returns a string broken up into lines.
+    /// An array of stirngs, each representing a line from an original string.
```
Comment (twostraws) [inline Unwrap/Extensions/String-Lines.swift]: "stirngs" should be corrected.

**Comment by twostraws** [comment]
Thank you for your contribution! It's generally better to submit several smaller pull requests that address specific issues, rather than submit one thing that does a lot at once. In this case, I don't wish to accept your change from triple to double comments, because I think it does more harm than good, but I can't accept the others until you with draw those.

It doesn't really matter whether triple-slashed comments are ignored or not; they are used consistently before things to document what they do. If some things are double-slashed and others tripled, it raises questions that really aren't interesting.

File: Unwrap/MainTabBarController.swift [hunk 2]
```diff
@@ -22,19 +22,20 @@ class MainTabBarController: UITabBarController, Storyboarded {
...
     func handle(_ launchOptions: [UIApplication.LaunchOptionsKey: Any]?) {
         if let item = launchOptions?[.shortcutItem] as? UIApplicationShortcutItem {
             handle(shortcutItem: item)
         }
     }
 
     func handle(shortcutItem: UIApplicationShortcutItem) {
-        if shortcutItem.type == "com.hackingwithswift.unwrapswift.challenges" {
+        switch shortcutItem.type {
+        case "com.hackingwithswift.unwrapswift.challenges":
```
Comment (Prabhakar-iOS) [inline Unwrap/MainTabBarController.swift]: Could be better to use enum

Comment (ramzesenok) [reply Unwrap/MainTabBarController.swift]: True, but that requires to create it, my idea was to optimise existing code

Comment (Prabhakar-iOS) [reply Unwrap/MainTabBarController.swift]: Okay.

**Comment by twostraws** [comment]
Great – thank you!

**Pull request merged**